### PR TITLE
refactor(qty-num): review editorially

### PR DESCRIPTION
**Pull request by JohelEGP**

### CODE REVIEW

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 1]
```diff
@@ -1301,7 +1301,7 @@ template<Dimension Lhs, Dimension Rhs>
 ### Quantity types
 
 Equality for quantity types is similar to dimensions. Again, users are allowed to derive their own
-types but only from the named strong types provided by the library:
+types:
```
Comment (JohelEGP) [inline src/2982R1_std_quantity_as_a_numeric_type.md:1304]: Maybe this one is wrong.
Can you confirm?

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 2]
```diff
@@ -1512,14 +1512,12 @@ std::cout << q1.in(m) << '\n';
...
+The second line above converts the input quantity to be expressed in metres and prints its
+value. The third line converts the quantity expressed in kilometers into one measured in
 metres.
 
-In both cases we assume that one can convert a quantity into another one with a unit of a higher
-resolution. There is no protection against overflow of the representation type. In case the target
-quantity ends up with a value bigger than the representation type can handle, we will be facing
-Undefined Behavior.
+In both cases, it is assumed that we can convert a quantity into another one with a unit of a higher
+resolution. There is no protection against overflow of the representation type built into the library.
```
Comment (JohelEGP) [inline src/2982R1_std_quantity_as_a_numeric_type.md:1520]: Removes the same text as commit 576418066354fd6af8f46af51726c9dd369c8f79.

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 3]
```diff
@@ -1791,18 +1789,18 @@ one:
...
-the division of two `std::chrono::duration` objects.
+the division of two `std::chrono::duration` values.
 
 In the initial design of this library, the resulting type of division of two quantities was their
-common representation type (the same as in case of `std::chrono::duration`):
+common representation type (just like `std::chrono::duration`):
 
 ```cpp
-static_assert(std::is_same_v<decltype(10 * km / (5 * km)), std::int64_t>);
+static_assert(std::is_same_v<decltype(10 * km / (5 * km)), int>);
```
Comment (JohelEGP) [inline src/2982R1_std_quantity_as_a_numeric_type.md:1803]: It's either this,
or using UDLs,
like in the old design.

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 4]
```diff
@@ -1907,13 +1907,14 @@ inline constexpr struct hubble_constant :
...
+- [@ISO80000] (part 3) provides a `rotation` quantity defined as the number of revolutions,
+- [@ISO80000] (part 6) provides a `number_of_turns_in_a_winding` quantity,
+- [@ISO80000] (part 13) provides a `Hamming_distance` quantity defined as the number of digit positions
   in which the corresponding digits of two words of the same length are different.
 
-Thanks to assigning strong names to such quantities, later on they can be explicitly used as
-arguments in the quantity equations of other quantities deriving from them.
+Thanks to assigning strong names to such quantities,
+they can be used in the quantity equation of other quantities.
+For example, `rotational_frequency` is defined by `rotation / duration`.
```
Comment (JohelEGP) [inline src/2982R1_std_quantity_as_a_numeric_type.md:1917]: "Deriving from them" is simply too confusing.
It's not the C++ term related to base and derived classes.
It's more like "they're used in the definition of new quantities".

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 5]
```diff
@@ -2066,7 +2067,7 @@ static_assert(isq::height(2 * m) - isq::height(1 * m) == isq::height(1 * m));
 
 The above uses the same types for LHS, RHS, and the result, but in general, we can add, subtract,
 or compare the values of any quantity type as long as both quantities are of the same kind.
-The result of such an operation will be the common type of the arguments:
+The result of addition and subtraction will be the common type of the arguments:
```
Comment (JohelEGP) [inline src/2982R1_std_quantity_as_a_numeric_type.md:2070]: Reading this now, I suppose "such" refers to those in the example,
as might be implied by the colon at the end.
My original read included the "compare" of the previous sentence,
which returns a `bool` or comparison category type, and not a quantity type.

File: src/2982R1_std_quantity_as_a_numeric_type.md [hunk 6]
```diff
@@ -2557,7 +2558,7 @@ Doing so has two important benefits:
    the thing being returned there.
 2. Such a concept constrains the type returned from the function. This means that it works as
    a unit test to verify if our function actually performs what it is supposed to do. If there is
-   an error in quantity equations, we will learn about it right away.
+   an error in quantity expressions, we will learn about it right away.
```
Comment (mpusz) [inline src/2982R1_std_quantity_as_a_numeric_type.md:2561]: This is incorrect. The correct term is "quantity equation" (https://jcgm.bipm.org/vim/en/1.22.html),

**Review by mpusz — CHANGES_REQUESTED** [review]
Thanks a lot!

File: src/2982R1_std_quantity_as_a_numeric_type.md (hunk 6 above)
Comment (JohelEGP) [reply src/2982R1_std_quantity_as_a_numeric_type.md:2561]: Hmm... Yeah, I think that makes sense.
The thing is that the equal sign of the equation is invisible in code,
so I thought it was an expression (<https://en.wikipedia.org/wiki/Expression_(mathematics)>).
In the case of `struct second : ... { } second;`,
the "assignment" of the expression in the base type list to the name `second` makes an equation.

**Review by mpusz — APPROVED** [review]
Thanks!

**Pull request merged**